jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml

```diff
@@ -490,6 +490,12 @@
         frequency: 'H H/6 * * *'
         trigger-job: ''
 
+    # kubeadm-gce
+    - kubernetes-e2e-kubeadm-gce:
```

Comment:
Please move to `bootstrap-ci-repo.yaml` and set `repo-name: k8s.io/kubernetes-anywhere`. Note that this will move the path to $GOPATH/k8s.io/kubernetes-anywhere instead of $WORKSPACE